addons/crm/tests/test_crm_pls.py

```diff
@@ -164,9 +164,10 @@ def test_predictive_lead_scoring(self):
             self._get_lead_values(False, 'no_team_%s' % str(13), country_ids[0], state_ids[1], state_values[2], state_values[0], source_ids[2], stage_ids[1]))
 
         leads = Lead.create(leads_to_create)
+        Lead.sudo().with_context(active_test=False).search([('id', 'not in', leads.ids)]).unlink()
```

Comment:
Shouldn't we move it into the setupClass so that all tests are data independent (in case we want to add some, tune some other, ...) ?